src/mlpack/methods/dbscan/dbscan_impl.hpp

```diff
@@ -100,7 +100,8 @@ size_t DBSCAN<RangeSearchType, PointSelectionPolicy>::Cluster(
     arma::Row<size_t>& assignments)
 {
   // Initialize the UnionFind object.
-  UnionFind uf(data.n_cols);
+  // The last element represents noise.
+  UnionFind uf(data.n_cols + 1);
```

Comment:
The last element of this "data.n_cols + 1" sized UnionFind representing noise seems unused.

Reply:
Yes, thank you!  This was a change I made during a draft, and forgot to change it back.  Reverted in c8d7a9575b.